src/ioctls/vm.rs

```diff
@@ -1799,6 +1799,23 @@ mod tests {
         assert!(vm.set_gsi_routing(&irq_routing).is_ok());
     }
 
+    #[test]
+    fn create_vcpu_different_cpuids() {
```

Comment:
nit: Can you rename this to `create_vcpu_different_ids`? We're using `cpuid` for something else.

Reply:
done!